testing/ui-tests/ui-test-users-id-tasks.js

```diff
@@ -0,0 +1,15 @@
+const puppeteer = require('puppeteer');
+
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.goto('https://nscstrdevusw2tuecommon.z5.web.core.windows.net/users/1/tasks');
+    
+  let date = new Date();
+  let stringDate = date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate() + '-' + date.getHours() + ':' + date.getMinutes() + ':' + date.getSeconds();
+
+  await page.screenshot({ path: './results/capture-users-id-tasks-'+ stringDate +'.png' });
```

Comment:
removing "capture" would be easier to read

Reply:
This is the naming convention Toddy specified in class. See below.
https://github.com/North-Seattle-College/ad440-winter2021-tuesday-repo/issues/214
The deliverable is a GitHub action that performs the task and saves the screenshot in Azure Storage container.
Naming conventions:

GitHub Action: Headless Chrome /users/id
Storage Account: nscstrdevusw2tuecommon
Container: screenshots
Filename Pattern: capture-users-id-YYYY-mm-DD-HH-MM-SS.png